### PR TITLE
An mpr with double64

### PR DESCRIPTION
**Pull request by AndreaNeumayr**
- enable DoubleFloats.jl package
- use `nepsMPR()` for mpr algorithm
- use `neps` for all other situations and remove local `neps`

### CODE REVIEW

File: src/Basics/constantsAndFunctions.jl [hunk 1]
```diff
@@ -7,19 +7,21 @@
 
 
 # Epsilon and sign
-const neps = sqrt( eps(Modia3D.MPRFloatType) )
+const neps = sqrt( eps() )
 
-function sign_eps(value::T; ) where {T}
-    seps::T = 100.0*neps
+nepsMPR(::Type{T}) where {T} = sqrt( eps(T) )
```
Comment (GerhardHippmann) [inline src/Basics/constantsAndFunctions.jl:12]: Function name should not include "MPR" because function is also used non MPR code.

Comment (AndreaNeumayr) [reply src/Basics/constantsAndFunctions.jl:12]: I changed that. In MPR code, including bounding boxes only `nepsMPR()` is used. Everything else uses `Modia3D.neps`

Comment (GerhardHippmann) [reply src/Basics/constantsAndFunctions.jl:12]: But isn't bounding box computation independent of MPR?

Comment (AndreaNeumayr) [reply src/Basics/constantsAndFunctions.jl:12]: There is the following issue: both, bounding box computation and MPR computation use support point calculation. Therefore, both call e.g. `supportPoint_abs_Cylinder`. Maybe we should discuss this offline?

Comment (GerhardHippmann) [reply src/Basics/constantsAndFunctions.jl:12]: That is the point: When support point calculation is called from bounding box computation, nepsMPR is misleading. You could rename it to nepsType or so.

Comment (AndreaNeumayr) [reply src/Basics/constantsAndFunctions.jl:12]: Ok, I'll change that.

**Pull request merged**